docs/development-instructions.rst

```diff
@@ -1,30 +1,148 @@
+      we have to see how that works in Windows, to see what happen when we have "Infineon" and "infineon" in the same path, or they are considered
+      the same. 
+      That arduino-cli naming will be more consistent with the snake_case naming conventions and non-redundant:
+      - arduino-cli compile --fqbn infineon:psoc:cy8ckit_062s2_ai 
+      instead of
+      - arduino-cli compile -fqbn Infineon-psoc:psoc:cy8ckit_062s2_ai 
+      or
+      - arduino-cli compile -fqbn Infineon:psoc:cy8ckit_062s2_ai
+
+4. Remove any existing installed "x.y.z" version of the core
+
+   ::
+
+      rm -rf x.y.z
+
+   .. This won´t be needed if the package is NOT installed using the .json package index installation.
+
+5. | Clone this repo in a folder with a semver version name. A orderly practice could be to bump the existing version, and add a suffix to indicate the feature or bugfix.
+   | For example, from installed ``1.1.0`` to ``1.2.0-new-feature``:
+
+   ::
+
+      git clone https://github.com/Infineon/arduino-core-psoc.git 1.2.0-new-feature
+
+   You can check if the correct version is installed by running
+
+   ::
+
+      arduino-cli core list
+
+   or in the Arduino IDE, navigate to *Tools > Board > Boards Manager...* and search for *PSoC*.
+
+   .. warning::
+      | The name of the repo directory needs to be a valid semver (x.y.z) version number. 
+      | For more information about the valid Arduino semver notation, see the `Package index specification - Platforms definitions <https://arduino.github.io/arduino-cli/0.34/package_index_json-specification/#platforms-definitions>`_.
+
+6. From the root directory of the core, run the setup script |:computer:|:
+
+   ::
+
+      bash tools/dev-setup.sh
+
+   .. note::
+      Alternatively, you can setup the development environment manually |:wrench:| following the steps in :ref:`Manual setup <dev_manual_setup>`.
+
+7. If using the Arduino IDE, restart the application.
+
+8. Start developing |:tools:|!
+
+
+.. _dev_manual_setup:
+
+Manual setup 
+------------
+
+| This section describes the manual setup of the development environment for the Infineon PSoC™ microcontroller core.
+| You can consider this a detailed explanation of the development setup process. 
+| Unless you want to know the details |:detective:|, it is easier and recommended to use the automated setup script |:scroll:| in :ref:`Environment setup <env_dev_setup>`.
+
+Installing the ArduinoCore-API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Locate in your computer where the Arduino15 packages are installed. By default:
+
+   ::
+
+      ~/.arduino15/
+
+   .. TODO: I would postpone the Windows part, as we won´t have all the dev tools available there
+            c:/Users/%USERNAME%/AppData/local/Arduino15 
+
+2. Change directories to the following path, where your ``arduino-core-psoc`` repository should be cloned:
+
+   ::
+
+      cd packages/Infineon-psoc/hardware/psoc/<x.y.z>
+
+3. 0. Initialize the ArduinoCore-API submodule
```

Comment:
You have to just remove one "0." here :)

Reply:
done! Sorry :)